core/validators.py

```diff
@@ -7,3 +9,11 @@ def validate_approximatedate(date):
             'Event date can\'t be a year only. '
             'Please, provide at least a month and a year.'
         )
+
+
+def validate_event_date(date):
+    today = datetime.today()
+    event_date = datetime.date(datetime.strptime('{0}-{1}-{2}'.format(date.year, date.month, date.day), '%Y-%m-%d'))
```

Comment:
This looks odd. Is this forcing a datetime or date object to a date? If it's already an instance of `date`, it shouldn't matter. If it is a datetime, `date.date()` would be the best way.